src/main/kotlin/com/cjbooms/fabrikt/model/SourceApi.kt

```diff
@@ -39,22 +43,40 @@ data class SourceApi(
         validateSchemaObjects(openApi3).let {
             if (it.isNotEmpty()) throw ParameterException("Invalid models or api file:\n${it.joinToString("\n\t")}")
         }
-        val globalSchemas = openApi3.schemas.entries.map { it.key to it.value }
-        val globalParameters = openApi3.parameters.entries.map { it.key to it.value.schema }
-        val globalRequests = openApi3.requestBodies.entries.flatMap {  request ->
+
+        val componentSchemas = openApi3.schemas.entries.map { it.key to it.value }
```

Comment:
**praise:** I see now why you chose to define explicit variables like this!
It seems not very scalable to me, but perhaps that is not required. I wonder if there is a solution which looks less repetitive but I cannot think of any. 
Well done. :)

Reply:
thank you for the explanation